### ISSUE TITLE
Unable to select Device from Management > Graphs screen

### ISSUE THREAD

**Issue by datatecuk** [Cacti/cacti#3875]
# Unable to select Device from the dropdown list in "Management > Graphs"

## To Reproduce

Steps to reproduce the behaviour:

1. Go to 'Management > Graphs'

2. Click on 'Device Dropdown List'

3. Try and select a device from the list and nothing happens.

## Expected behaviour

When you select a device from the list it should select the device and update the screen with all graphs that have been created for that device.

## Desktop

- OS: Windows 10

- Browser Chrome, Safari, Edge

## Additional context

The template dropdown works fine in the same screen.

**Comment by cigamit** [comment]
What version of Cacti?  This is working for me on Windows 10 / Chrome with the latest version from github devel branch.

**Comment by datatecuk** [comment]
latest v1.2.15 branch in GitHub

**Comment by TheWitness** [comment]
Be specific in what you mean can not select.  Is it simply missing from the list?  This dropdown is an auto-complete, so the easiest way to find the host is to simply clear the value and start typing the hostname.

**Comment by datatecuk** [comment]
I'm not able to select any of the devices from the "Devices" dropdown list on the Graphs page under Management
![image](https://user-images.githubusercontent.com/5603694/96722404-400c1f80-13a5-11eb-93b0-09ef7d2419cf.png)

**Comment by netniV** [comment]
I have just pulled the latest version to the office and that is working correctly in all browsers.  Can you check that you are not having console errors? If you are, can you post them please.

**Comment by datatecuk** [comment]
Just checked and the following console error appears when you try and select a device from the dropdown list.

```
main.js?03e801189e783168a10b02de519a079b:154 Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'  'unsafe-inline' ".

    at HTMLInputElement.select (main.js?03e801189e783168a10b02de519a079b:154)
    at $.<computed>.<computed>._trigger (jquery-ui.js?f1f075fe32aca8216e55c5d00106fba7:697)
    at $.<computed>.<computed>.menuselect (jquery-ui.js?f1f075fe32aca8216e55c5d00106fba7:8273)
    at HTMLUListElement.handlerProxy (jquery-ui.js?f1f075fe32aca8216e55c5d00106fba7:606)
    at HTMLUListElement.dispatch (jquery.js?23c7c5d2d1317508e807a6c7f777d6ed:5429)
    at HTMLUListElement.elemData.handle (jquery.js?23c7c5d2d1317508e807a6c7f777d6ed:5233)
    at Object.trigger (jquery.js?23c7c5d2d1317508e807a6c7f777d6ed:8715)
    at HTMLUListElement.<anonymous> (jquery.js?23c7c5d2d1317508e807a6c7f777d6ed:8793)
    at Function.each (jquery.js?23c7c5d2d1317508e807a6c7f777d6ed:381)
    at jQuery.fn.init.each (jquery.js?23c7c5d2d1317508e807a6c7f777d6ed:203)
```

![image](https://user-images.githubusercontent.com/5603694/96856241-989af580-1455-11eb-9ff9-7ee76722b5d9.png)

**Comment by TheWitness** [comment]
Looks like a browser config issue.  Also looks like develop, unless you created that theme yourself.

**Comment by datatecuk** [comment]
I copied over the "cacti" theme from the development branch as it looks great.
I have just reverted the theme back to the "Modern" theme and it works fine now, so it is a theme issue.

Can the "Cacti" theme be included in the 1.2.x branch?

**Comment by TheWitness** [comment]
I'll leave that to the theme author to decide.
